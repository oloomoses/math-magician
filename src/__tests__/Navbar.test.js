import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar component', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Router><Navbar /></Router>);
    expect(tree).toMatchSnapshot();
  });
});
