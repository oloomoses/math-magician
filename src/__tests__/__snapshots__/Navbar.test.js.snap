// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component matches the snapshot 1`] = `
<div
  className="navbar"
>
  <div
    className="logo"
  >
    Math Magician
  </div>
  <div
    className="nav"
  >
    <div
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </div>
    <div
      className="separator"
    >
      |
    </div>
    <div
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </div>
    <div
      className="separator"
    >
      |
    </div>
    <div
      className="nav-item"
    >
      <a
        className="nav-link"
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </div>
  </div>
</div>
`;
